src/index.html

```diff
@@ -1,13 +1,59 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css" >
+  </head>
+  <body>
+    <div class="product">
+      <div class="product__card" data-qa="card">
+        <div class="product__image">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+            class="product__image product__image--image"
```

Comment:
Is this "product__image--image" class is modifier for element "product__image"? What is it changing? Or maybe you've meant something else? See if you correctly name classes in this document.
https://mate-academy.github.io/fe-program/css/typical-bem-mistakes